Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+from typing import Union, Dict, Optional, Any
+
+import demistomock as demisto
+import requests
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# GLOBALS
+CB_ORG_KEY: str
+
+
+def convert_unix_to_timestamp(timestamp):
+    """
+    Convert millise since epoch to date formatted MM/DD/YYYYTHH:MI:SS
+    """
+    if timestamp:
+        date_time = datetime.utcfromtimestamp(timestamp / 1000)
+        return date_time.strftime('%Y-%m-%dT%H:%M:%SZ')
+    return ''
+
+
+def convert_date_to_unix(dstr):
+    """
+    Convert a given string with MM/DD/YYYY format to millis since epoch
+    """
+    dt = datetime.strptime(dstr, '%Y-%m-%dT%H:%M:%SZ')
+    return dt.timestamp()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+    def __init__(self, base_url: str, use_ssl: bool, use_proxy: bool, token=None):
+        self.token = token
+        super().__init__(base_url, verify=use_ssl, proxy=use_proxy, headers={'Accept': 'application/json',
+                                                                             'Content-Type': 'application/json'})
+        if self.token:
+            self._headers.update({'X-Auth-Token': self.token})
+
+    def test_module_request(self):
+        url_suffix = f'/appservices/v6/orgs/{CB_ORG_KEY}/alerts/_search'
+        body = {
+            "criteria": {
+                "group_results": True,
+                "minimum_severity": 3
+            },
+            "sort": [{"field": "first_event_time", "order": "DESC"}],
+            "rows": 1,
+            "start": 0
+        }
+
+        return self._http_request('POST', url_suffix=url_suffix, json_data=body)
+
+    def search_alerts_request(self, group_results: bool, minimum_severity: int, create_time: dict,
+                              device_os_version: str, policy_id: int, alert_tag: str, alert_id: str,
+                              device_username: str, device_id: int, device_os: str, process_sha256: str,
+                              policy_name: str, reputation: str, alert_type: str, alert_category: str, workflow: str,
```

Comment:
most of the type hints here are wrong
most should be `List` and `limit` is a `str`